### PR TITLE
update assimp to v5.2.3

### PR DESCRIPTION
**Pull request by 45degree**
update assimp to v5.2.3

### CODE REVIEW

File: packages/a/assimp/xmake.lua [hunk 1]
```diff
@@ -77,6 +71,15 @@ package("assimp")
         add_config_arg("no_export",        "ASSIMP_NO_EXPORT")
         add_config_arg("asan",             "ASSIMP_ASAN")
         add_config_arg("ubsan",            "ASSIMP_UBSAN")
+
+        -- since v5.3.2, assimp allow user to choose C runtime library version, on Windows
+        -- https://github.com/assimp/assimp/pull/4444
+        if package:version():ge("v5.2.3") then
+            if package:is_plat("windows") and package:config("vs_runtime") == "MT" then
```
Comment (SirLynix) [inline packages/a/assimp/xmake.lua:78]: You should test `"MTd"` too.

**Comment by SirLynix** [comment]
assimp last version broke MTd/MDd packages (see https://github.com/assimp/assimp/pull/4444 )
I think we have to patch CMakeLists.txt to get rid of this:
```
    if (${USE_STATIC_CRT})
      add_compile_options(
          $<$<CONFIG:>:/MT>
          $<$<CONFIG:Debug>:/MTd>
          $<$<CONFIG:Release>:/MT>
      )
    else()
      add_compile_options(
          $<$<CONFIG:>:/MD>
          $<$<CONFIG:Debug>:/MDd>
          $<$<CONFIG:Release>:/MD>
      )
    endif()
```

**Comment by 45degree** [comment]
Just delete this snippet in CMakeList.txt ?

**Comment by SirLynix** [comment]
Yes, we need the package to patch it out.

**Comment by 45degree** [comment]
OK, I submitted a new PR

**Pull request merged**